docs/UserGuide.adoc

```diff
@@ -34,84 +37,152 @@ THRIFT is for money-conscious NUS students who wish to track their spending, in
-*   Parameters can be in any order e.g. if the command specifies `n/EXPENSE_NAME v/VALUE`, `v/VALUE n/EXPENSE_NAME` will achieve the same outcome.
+Understanding this document:
+[horizontal]
+`add_expense`:: Words with a grey highlight (called a mark-up) indicates that it is a command, or a part of it.
+kbd:[Enter]:: Words styled like this indicates a keyboard button.
+UPPER_CASE:: Words in UPPER_CASE are parameters to be supplied by you. e.g. `add_expense n/EXPENSE_NAME v/VALUE`.
+NOTE: Parameters can be in any order e.g. if the command specifies `n/EXPENSE_NAME v/VALUE`, `v/VALUE n/EXPENSE_NAME` will achieve the same outcome.
+**add_expense n/Lunch v/5**:: Words in bold indicates the input that would be entered by you.
+_Debt_:: Words in italics indicate the literals used in the example commands.
+[ ] :: Words in square brackets `[ ]` are optional e.g. `n/EXPENSE_NAME [t/TAG]` can be used as `n/Laksa`.
+( | ) :: Words in round brackets `( )` with `|` are mandatory parameters where a selection is required e.g. `tag (add | delete)` means you need to input either `tag add` or `tag delete`.
+... ::   Parameters with `...` after them can be used as often as zero to multiple times. e.g. `[t/TAG] …` can be used as ` ` (i.e. 0 times), `t/Food t/Dinner` etc.
 
 [[BudgetTag]]
 === Setting monthly budget: `budget`
 
-Format: `budget v/VALUE d/MONTH`, where `MONTH` are digits in `MM/YYYY`.
+You are able to set a budget for each month in THRIFT. This allows THRIFT to display your remaining balance for the month and lets you keep your spending in check.
 
-Example:
+NOTE: It is not compulsory to set a budget for each month and you can use THRIFT normally without setting budgets.
 
-*   `budget v/1000 d/10/2019`
+==== Command syntax:
+`budget v/VALUE d/MONTH`
 
-Expected output:
+****
+* Sets your budget for the month specified in `MONTH` as `VALUE`.
+* `VALUE` is any positive integer below one billion.
+* `MONTH` is in `MM/YYYY` and `MM` is between `01-12`.
+****
+
+==== Example usage:
 
-*   `New budget: +
-Amount: $1,000.00 +
-Month: 10/2019`
+You wish to set your budget for the month _October 2019_ as $_2000_:
 
+. Your current budget for the month _October 2019_ is $_0_.
++
+image::/images/add-budget/Step1.png[,550]
+
+. You type: **budget v/2000 d/10/2019** in the command box and press kbd:[Enter].
++
+image::/images/add-budget/Step2.png[,550]
+
+. You should see in the result box that your budget for _October 2019_ has been set to $_2000_.
++
+image::/images/add-budget/Step3.png[,550]
+
+NOTE: If you are changing your budget for the current displayed month, you should see the changes on the GUI immediately.
 
 [[ExpenseTag]]
 === Adding an expense: `add_expense`
 
-Format: `add_expense n/EXPENSE_NAME v/VALUE [r/REMARKS] [t/TAG]...`
+One of the most important feature of THRIFT is to allow you to track your expenses by adding them into THRIFT.
```

Comment:
Should be "one of the most important **features**". Not to nitpick but Diana will notice :(

Reply:
Fixed! Please point as many of these out as we all want to perfect our documents :laughing: